internal/workflow/processing_test.go

```diff
@@ -482,6 +502,42 @@ func (s *ProcessingWorkflowTestSuite) TestAutoApprovedAIP() {
 		nil,
 	)
 
+	s.env.OnActivity(
+		createPreservationTaskLocalActivity,
+		ctx,
+		&createPreservationTaskLocalActivityParams{
+			PkgSvc: pkgsvc,
+			RNG:    s.workflow.rng,
+			PreservationTask: datatypes.PreservationTask{
+				Name:   "Validate PREMIS",
+				Status: enums.PreservationTaskStatusInProgress,
+				StartedAt: sql.NullTime{
+					Time:  startTime,
+					Valid: true,
+				},
+				PreservationActionID: 0,
```

Comment:
It would be nice to create a variable (`paID = 1`) at the start of the test, return that var instead of `0` from the create preservation action activity and reuse that var and all the create preservation task calls.

Reply:
Thanks for doing across all tests.